mlir/test/Target/LLVMIR/omptarget-llvm.mlir

```diff
@@ -441,3 +441,41 @@ llvm.func @_QPopenmp_target_use_dev_both() {
 // CHECK:         ret void
 
 // -----
+
+llvm.func @_QPopenmp_target_data_update() {
```

Comment:
Could you add more tests to cover different data types (like Fortran pointers, allocatable arrays etc.)

Reply:
As discussed offline, the FIR dialect is not registered with `mlir-translate`. So we cannot use `fir.ref` in `omptarget-llvm.mlir`.

However, I will double check with @agozillon whether we have existing tests for lowering `map_info` op's arguments from `fir` to `llvm` types or not. If not, I will open another PR with more testing.